.appveyor.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration: <https://www.appveyor.com/docs/build-configuration/>
+# Image software: <https://www.appveyor.com/docs/windows-images-software/>
+# Script reference: <https://www.appveyor.com/docs/appveyor-yml/>
+# Environment variables: <https://www.appveyor.com/docs/environment-variables/>
+
+# Platform
+image: Visual Studio 2019 # Windows Server 2016 + MSVC 2019 toolchain
+
+# Source
+branches:
+  only:
+    - master
+
+# Environment variables
+environment:
+  PYTHON32: C:\Python37
+  PYTHON64: C:\Python37-x64
+
+# Add dependencies
+install:
+  # VSRepo
+  - git clone https://github.com/vapoursynth/vsrepo
+  # zimg 2.9.x
+  - git clone https://github.com/sekrit-twc/zimg --branch v2.9
+  # Avisynth+ (MT)
+  - git clone https://github.com/pinterf/AviSynthPlus --branch MT
+  # Python3
+  - if "%PLATFORM%" == "Win32" (set "PATH=%PYTHON32%;%PYTHON32%\Scripts;%PATH%") else (set "PATH=%PYTHON64%;%PYTHON64%\Scripts;%PATH%")
+  - python --version
+  # Python environment
+  - python -m pip install -r python-requirements.txt
+  # 7zip
+  - 7z --help
+
+# Build
+configuration: Release
+platform:
+  - Win32
+  - x64
+build:
+  project: msvc_project/VapourSynth.sln
+  verbosity: minimal
+after_build:
+  # Python wrapper
+  - call cython_build.bat
+  # Documentation
+  - call docs_build.bat
+  # !!! No portable package or installer because their build scripts are trash !!!
```

Comment:
ooookay there, dude

Reply:
i mean it might very well be true as far as i know but it's, uh, a somewhat unusual place to make that comment

Reply:
Yeah, I think you're gonna need this to get this merged, I think, Youka.

Reply:
Can you please build at least the portable version as a built-artifact.
Also list the content of the dist-folder for x32 and x64 as generated by the command `python setup.py bdist_wheel sdist --format=zip` on both architectures?

Reply:
* WHL files get to **compile_output** artifact ([commit](https://github.com/vapoursynth/vapoursynth/pull/478/commits/e8ff88a4e10bc2bb5da56e35b5d16145f8a049fb)).
* Packaging **portable release** was added as artifact: [successful build](https://ci.appveyor.com/project/Youka/vapoursynth/builds/26621505) ([commit](https://github.com/vapoursynth/vapoursynth/pull/478/commits/b177ef407656b05d21cd13072c49e8bade70c8d6)).